docs/cheatsheet/admin.rst

```diff
@@ -0,0 +1,32 @@
+.. _ref_cheatsheet_admin:
+
+Admin Commands
+==============
+
+Create a database:
+
+.. code-block:: edgeql-repl
+
+    db> CREATE DATABASE my_new_project;
+    CREATE
+
+Create a role:
+
+.. code-block:: edgeql-repl
+
+    db> CREATE SUPERUSER ROLE project;
```

Comment:
Add an an example of configuring Auth

Reply:
```
CONFIGURE SYSTEM INSERT Auth {
    priority := 0,
    method := (INSERT Trust),
}
```
Do we actually have any other ones? This one is just passwordless access, right?

Reply:
@elprans

Reply:
We also have `SCRAM` for password logins.

Reply:
Added more examples.